vendingMachine.js

```diff
@@ -13,34 +13,50 @@ let stock = [
 function showVendingMachine(arr1, arr2) {
   let result = '';

```

Comment:
All of these 'variables' change their value? what about using const?

Reply:
Some of these variables do change their value, the use of const was applied for some variables whose value is fixed.